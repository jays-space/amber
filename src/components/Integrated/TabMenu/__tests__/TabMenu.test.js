import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { TabMenu } from "..";

describe("<TabMenu />", () => {
  it("should render correctly", () => {
    render(<TabMenu />);

    const tabMenuComponent = screen.getByTitle("tab-menu");
    expect(tabMenuComponent).toBeInTheDocument();
    expect(tabMenuComponent).toMatchSnapshot();
  });
});
