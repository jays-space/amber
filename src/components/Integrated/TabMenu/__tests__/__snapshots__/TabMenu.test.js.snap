// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabMenu /> should render correctly 1`] = `
<div
  class="hidden lg:flex"
  title="tab-menu"
>
  <div
    class="px-6 py-4 cursor-pointer hover:bg-slate-50 transform ease-out delay-100 border-b-2 border-yellow-500"
  >
    <p
      class="tracking-wide  text-slate-700 font-bold text-sm uppercase"
      data-testid="paragraph"
    >
      details
    </p>
  </div>
  <div
    class="px-6 py-4 cursor-pointer hover:bg-slate-50 transform ease-out delay-100 false"
  >
    <p
      class="tracking-wide  text-slate-700 opacity-80 text-sm uppercase"
      data-testid="paragraph"
    >
      resume
    </p>
  </div>
  <div
    class="px-6 py-4 cursor-pointer hover:bg-slate-50 transform ease-out delay-100 false"
  >
    <p
      class="tracking-wide  text-slate-700 opacity-80 text-sm uppercase"
      data-testid="paragraph"
    >
      hiring pipeline
    </p>
  </div>
  <div
    class="px-6 py-4 cursor-pointer hover:bg-slate-50 transform ease-out delay-100 false"
  >
    <p
      class="tracking-wide  text-slate-700 opacity-80 text-sm uppercase"
      data-testid="paragraph"
    >
      interviews
    </p>
  </div>
</div>
`;
